extensions/empty-screenshots/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { readdir, stat, rm } from "fs/promises";
+import { join, resolve } from "path";
+import { homedir } from "os";
+import { showToast, Toast, showHUD, getPreferenceValues } from "@raycast/api";
+
+interface Preferences {
+  daysToKeep: string;
+  screenshotFolder: string;
+}
+
+export default async function main() {
+  const { daysToKeep, screenshotFolder } = getPreferenceValues<Preferences>();
+  const numberOfDays = parseInt(daysToKeep, 10);
+  const folder = resolve(screenshotFolder.replace("~", homedir));
+
+  const files = await readdir(folder);
+
+  const screenshots = files
+    .filter((file) => file.endsWith(".png") || file.endsWith(".mov"))
+    .map((file) => join(folder, file));
+
+  const today = new Date();
+  const priorDate = new Date(new Date().setDate(today.getDate() - numberOfDays));
+
+  await showToast({
+    style: Toast.Style.Animated,
+    title: `🗑 Cleaning ${numberOfDays} worth screenshots`,
```

Comment:
I'm not sure the sentence make sense?

Reply:
Ah, this used to be the number of screenshots it was going to try and delete, but changed it at some point! Good catch! Updated this to `🗑 Cleaning ${numberOfDays} days worth of screenshots`